deployment/common/common.yaml

```diff
@@ -21,4 +21,77 @@ spec:
   scope: Namespaced
   names:
     plural: routes
-    kind: Route
\ No newline at end of file
+    kind: Route
+  validation:
+    # openAPIV3Schema is the schema for validating custom objects.
+    openAPIV3Schema:
+      properties:
+        spec:
+          required:
```

Comment:
Should the routes property be required as well?

Reply:
yes can make that explicit, I think having the serviceName & servicePort underneath be required will require the route, but in any case I'll make it clear.